packages/caddy-ingress/v0.2.1+1/package.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "caddy-ingress-controller"
+shortDescription: "This is the Kubernetes Ingress Controller for Caddy"
+iconUrl: "https://avatars.githubusercontent.com/u/12955528"
+scope: Cluster
+defaultNamespace: "caddy-system"
+helm:
+  repositoryUrl: "https://caddyserver.github.io/ingress"
+  chartName: "caddy-ingress"
+  chartVersion: "0.2.1"
+valueDefinitions:
+  automaticHTTPS:
+    metadata:
+      description: Add you email address for automatic HTTPS - "your@email.com"
+    type: text
+    targets:
+      - chartName: caddy-ingress
+        patch:
+          op: add
+          path: /ingressController/config/email
```

Comment:
A current limitation of value definitions is that they can only patch helm values in existing paths. So, in this case, the values should be initially set to 
```yaml
ingressController:
  config: {}
```